Lib9c/Battle/RaidBoss.cs

```diff
@@ -11,9 +11,6 @@ namespace Nekoyume.Model
 {
     public class RaidBoss : Enemy
     {
-        [NonSerialized]
-        public RaidSimulator RaidSimulator;
-
```

Comment:
is this correct working without simulator?

Reply:
This field is initialized but not used.